CHANGES.md

```diff
@@ -5,5 +5,51 @@ First stable release.
 
 # grmtools 0.1.1 (2018-12-18)
 
-* Fix bug where %epp strings with quote marks in caused a code-generation
+* Fix bug where `%epp` strings with quote marks in caused a code-generation
   failure in compile-time mode.
+
+
+# grmtools 0.2.0 (2019-01-14)
+
+## Breaking changes
+
+* Previously we forced users to specify the `YaccKind` of a grammar and then the
+  `ActionKind` of actions. This is unnecessarily fiddly, so remove `ActionKind`
+  entirely and instead flesh out `YaccKind` to deal with the possible variants.
+  For example `ActionKind::CustomAction` is now, in essence,
+  `YaccKind::Original(YaccOriginalActionKind::UserAction)`. This is a breaking
+  change but one that will make future evolution much easier.
+
+* The `%type` directive in grammars exposed by
+  YaccKind::Original(YaccOriginalActionKind::UserAction) has been renamed to
+  `%actiontype` to make it clear *what* type is being referred to. In general,
+  we expect most people to move to the `YaccKind::Grmtools` variant (see below),
+  which doesn't require the `%actiontype` directive.
+
+## Major changes
+
+* grmtools ported to 2018 and thus needs rustc-1.31 or later to compile.
```

Comment:
> grmtools **was** ported

Reply:
Agreed, this was awkward. Fix below (hopefully).